tests/baselines/reference/multipleDefaultExports01.errors.txt

```diff
@@ -7,6 +7,7 @@ tests/cases/conformance/es6/modules/m1.ts(10,16): error TS2528: A module cannot
     export default class foo {
                          ~~~
 !!! error TS2528: A module cannot have multiple default exports.
+!!! related TS2730 tests/cases/conformance/es6/modules/m1.ts:1:22: This export conflicts with the first.

```

Comment:
that is because function bind first with `bindEachFunctionsFirst`,
seems expect behavior